ext/session/tests/session_set_save_handler_closures.phpt

```diff
@@ -19,7 +19,8 @@ var_dump(session_module_name(FALSE));
 var_dump(session_module_name("blah"));
 var_dump(session_module_name("foo"));
 
-$path = __DIR__;
```

Comment:
I think you are interested in it, @Girgias ;)

Reply:
*finally*